mingw-w64-git/PKGBUILD

```diff
@@ -4,7 +4,8 @@ _realname=git
 pkgbase="mingw-w64-${_realname}"
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}"
 	 "${MINGW_PACKAGE_PREFIX}-${_realname}-doc-html"
-	 "${MINGW_PACKAGE_PREFIX}-${_realname}-doc-man")
+	 "${MINGW_PACKAGE_PREFIX}-${_realname}-doc-man"
+	 "${MINGW_PACKAGE_PREFIX}-${_realname}-for-windows-addons")
```

Comment:
Nit: The doc packages continue to contain the docs for everything, even the addons.

Reply:
You're right! While it [does remove other packages' manual pages](https://github.com/git-for-windows/MINGW-packages/blob/b57f142c9b3cc30af1ff4dc62982625cf7d16461/mingw-w64-git/PKGBUILD#L220-L256), this PR does not yet handle `git-bash.1`/`git-bash.html` as intended. Will fix.